composer.json

```diff
@@ -8,8 +8,8 @@
         "http-flow"
     ],
     "require": {
-        "php": "^5.5|^7.0",
-        "nikic/fast-route": "~0.4",
+        "php": "^5.6|^7.0",
+        "nikic/fast-route": "~0.6.0",
```

Comment:
Including the patch part, this dependency can go only from `0.6.0` to `0.6.*`, I suggest to use `~0.6`.